### ISSUE TITLE
[Suggestion] Laying down/Standing up seat

### ISSUE THREAD

**Issue by Xerma** [tsunamayo/Starship-EVO#2334]
We used to have this before the visual update and it came to be very useful in various situations and permitted varying designs for cockpits and such. I would love to have this back!

![20180131113858_1](https://user-images.githubusercontent.com/32974671/83711907-2468ce00-a5ea-11ea-8eba-461df94c605d.jpg)

**Comment by Aiwii** [comment]
Yes! please add more seating positions

**Comment by ZachZent** [comment]
Especially for hovercraft

**Comment by Misteropie** [comment]
Belly / sport bike "seats" would be a nice addition as well! Something like this
![image](https://user-images.githubusercontent.com/66394949/83712456-b7017100-a5da-11ea-870f-9a652a5bd371.png)

**Comment by Dwarf-LordPangolin** [comment]
Agreed, particularly the motorcycle-style seat; an upright sitting position looks a bit funny on many hovercraft. :wink:

**Comment by chacalik** [comment]
I agree and for a motorcycle seat like, a 2 blocks long model should be fine, so we can make thinner hoverbikes, what will recall that the yoke is 2 blocks wide, than we would also require a 3 blocks wide

**Comment by Xenoprophet-Nail** [comment]
I have to agree, the smallest seat we have is technically the ottoman, it's hard to make a sleek aggressive bike with a round seat.. It was great on mt scooter, but on my race bike.. well: 
![20200525075708_1](https://user-images.githubusercontent.com/51257864/83736174-42045a80-a61f-11ea-9f24-059079865096.jpg) Hard to get a streamlined look without bulking it up a lot wouldn't you agree?

**Comment by Kaiser-Indrasil** [comment]
Yes please! Add motorbike seats! Much like Xeno's bike, mine uses the ottoman as well but the character model's legs should be apart, otherwise it's a scooter seat, not a motorcycle seat.
![Starship_EVO_naPvNsHDg1](https://user-images.githubusercontent.com/63756057/83856018-d8647900-a710-11ea-8bea-2cccbffb6e20.png)

**Comment by tsunamayo** [comment]
@Xerma and all, sorry yes it make a lot of sense, the reason I have been holding on this is I need to complete a IK refactoring for the character animation. Cheers
